Mesh_3/include/CGAL/make_mesh_3.h

```diff
@@ -401,6 +407,33 @@ struct C3t3_initializer < C3T3, MD, MC, true, CGAL::Tag_false >
  *                           </UL>}
  *     \cgalParamDefault{`parameters::exude()`}
  *   \cgalParamSectionEnd
+ *   \cgalParamSectionBegin{Mesh initialization with a functor}
+ *     \cgalParamDescription{an `InitialPointsGenerator` can optionally be provided to start the meshing process.
+ *                           It must follow the `InitialPointsGenerator` concept.
```

Comment:
```suggestion
 *     \cgalParamDescription{an `InitialPointsGenerator` can optionally be provided to start the meshing process.
```
It's said in the first sentence already